### PR TITLE
matchArgs improvements

### PR DESCRIPTION
**Pull request by tpapagian**
1. Support multiple filters in matchArgs
2. Fix path argument type

### CODE REVIEW

File: pkg/selectors/kernel.go [hunk 1]
```diff
@@ -395,9 +395,17 @@ func ParseMatchArg(k *KernelSelectorState, arg *v1alpha1.ArgSelector, sig []v1al
...
 func ParseMatchArgs(k *KernelSelectorState, args []v1alpha1.ArgSelector, sig []v1alpha1.KProbeArg) error {
+	actionOffset := GetCurrentOffset(k)
 	loff := AdvanceSelectorLength(k)
-	for _, a := range args {
+	argOff := make([]uint32, 5)
+	for i := 0; i < 5; i++ {
+		argOff[i] = AdvanceSelectorLength(k)
+		WriteSelectorOffsetUint32(k, argOff[i], 0)
+	}
+	for i, a := range args {
```
Comment (kkourt) [inline pkg/selectors/kernel.go:410]: Can we return an error if user species more than 5 arguments?

**Review by kkourt — APPROVED** [review]
LGTM, but let's return an error if user specifies more than the supported number of arguments.

File: pkg/selectors/kernel.go (hunk 1 above)
Comment (tpapagian) [reply pkg/selectors/kernel.go:417]: Done!

**Comment by jrfastab** [comment]
new PR here https://github.com/cilium/tetragon/pull/901

**Pull request closed without merging**